src/input/findReportedConfiguration.ts

```diff
@@ -1,7 +1,7 @@
 import { Exec } from "../adapters/exec";
 
 export type DeepPartial<T> = {
-    [P in keyof T]: T[P] extends {} ? DeepPartial<T[P]> : T[P];
+    [P in keyof T]?: T[P] extends Record<P, T[P]> ? DeepPartial<T[P]> : T[P];
```

Comment:
Thanks for the suggestion @bbarry! Seems to work great.

Reply:
Reading it more this type is wrong? I think it should work for now because it is basically being treated as `any` here but I'll try and write some tests and put in a pr at some point.